git-versions.yml

```diff
@@ -31,9 +31,6 @@ services:
     mender-deviceconfig:
         image: mendersoftware/deviceconfig:1.2.x
 
-    mender-reporting:
-        image: mendersoftware/reporting:master
-
```

Comment:
Removing this should be enough, the release tool should play well with the rest as long as we have `release_component: false`. 

So honestly I vote for restoring the rest and treat it just as a component not ready to be released.

Reply:
https://github.com/mendersoftware/integration/pull/1856/commits/e535ee8353a475c4d591c9341aee30ead4e7d32c